db/local/db.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+import os
+import subprocess
+from typing import Type
+
+from peewee_migrate import Router
+
+from src.base_model import BaseModel, db
+
+MIGRATION_DIRECTORY: str = "db/local/migrations"
+
+models: list[Type[BaseModel]] = BaseModel.__subclasses__()
```

Comment:
Is that how it is supposed to work? Seems kind of brittle...

Reply:
It works 😄 But we can think about declaring an array by hand. This would require changing the array, when declaring a new subclass of `BaseModel`.

Reply:
I guess we can leave it as-is.